src/bedriftsmeny/velger/Virksomhetsvelger.tsx

```diff
@@ -32,14 +100,11 @@ const Velger = () => {
     useEffect(() => {
         if (åpen) {
             valgtUnderenhetRef.current?.focus();
+        } else {
+            setSøketekst('')
+            velgOrganisasjonIntern(valgtOrganisasjon)
         }
-    }, [åpen]);
-
-    const antallTreff = aktivtOrganisasjonstre.reduce(
-        (totaltAntall, juridiskEnhet) => totaltAntall + juridiskEnhet.Underenheter.length,
-        0
-    );
-
+    }, [åpen, valgtUnderenhetIntern]);
```

Comment:
mulig det finnes en bedre måte, men jeg fikk et issue med at focus på ref gikk for tidlig on tab event. Endte derfor med å legge til en dependency her.